### PR TITLE
Create py.typed

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by NeilGirdhar** [comment]
This fixes my type annotation problems with chex:
```python
from chex import Array
import jax.numpy as jnp
 
x: Array
y: jnp.ndarray
 
reveal_type(x)  # note: Revealed type is 'Any'
reveal_type(y)  # note: Revealed type is 'jax._src.numpy.lax_numpy.ndarray'
```

**Review by hbq1 — APPROVED** [review]

**Comment by hbq1** [comment]
Nice catch! Thanks a lot for your contribution.

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 """Install script for setuptools."""
 
 import os
-from setuptools import find_namespace_packages
+from setuptools import find_packages
```
Comment (hbq1) [inline setup.py:18]: Is this change necessary? I'd rather keep find_namespace_packages if possible.

Comment (graingert) [reply setup.py:18]: yes, otherwise you need multiple `py.typed` throughout the source tree
